components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/device/grant/DeviceFlowGrantTest.java

```diff
@@ -47,14 +61,19 @@ public class DeviceFlowGrantTest {
     private Timestamp newTime = new Timestamp(date.getTime());
     private DeviceFlowDO deviceFlowDO1 = new DeviceFlowDO();
     private DeviceFlowDO deviceFlowDO2 = new DeviceFlowDO();
+    private DeviceFlowGrant deviceFlowGrant;
+    private OAuthTokenReqMessageContext oAuthTokenReqMessageContext;
+    private DeviceFlowDO deviceFlowDO3;
```

Comment:
cant we move near to other DOs

Reply:
Ideally, these initializations should happen only one time that is BeforeClass, the other Dos are placed in BeforeMethod which is unnecessary. We need to fix those issues when adding the unit tests fro the whole flow.